zebrad/tests/acceptance.rs

```diff
@@ -47,6 +47,7 @@ const LAUNCH_DELAY: Duration = Duration::from_secs(10);
 fn default_test_config() -> Result<ZebradConfig> {
     let auto_port_ipv4_local = zebra_network::Config {
         listen_addr: "127.0.0.1:0".parse()?,
+        new_peer_interval: Duration::from_secs(30),
```

Comment:
Conflict with #1855:
```suggestion
        crawl_new_peer_interval: Duration::from_secs(30),
```